client/source/components/RR/ReviewTile.jsx

```diff
@@ -60,66 +52,71 @@ const ReviewTile = function ReviewTile({ review, updateList }) {
   };
 
   return (
-    <div className="review-tile">
-      <div className="review-header">
-        <span className="review-rating">{convertRating(review.rating)}</span>
-        <span className="review-user-date">
-          {`${review.reviewer_name}, `}
-          {formattedDate}
-        </span>
-      </div>
-      <div className="review-summary">{capSummary(review.summary)}</div>
-      <div className="review-body-container">
-        <div className="review-text">{capBody(review.body)}</div>
-        { review.body.length > 250 && fullBody === false
+    <ReviewTileContainer>
+      <div className="review-tile">
+        <div className="review-header">
+          <ReviewRating rating={review.rating} />
+          <div className="review-user-date">
+            {`${review.reviewer_name}, `}
+            {formattedDate}
+          </div>
+        </div>
+        <div className="review-summary">{capSummary(review.summary)}</div>
+        <div className="review-body-container">
+          <div className="review-text">{capBody(review.body)}</div>
+          { review.body.length > 250 && fullBody === false
+            ? (
+              <button
+                className="show-more-button"
+                type="button"
+                onClick={() => setFullBody(true)}
+              >SHOW MORE +
+              </button>
+            )
+            : null }
+          <ReviewPhotos>
+            <div className="review-photos">
+              {review.photos.map((photo) => (
+                <ReviewPhoto
+                  photo={photo}
+                  key={photo.id}
+                />
+              ))}
+            </div>
+          </ReviewPhotos>
+        </div>
+        { review.recommend
+          ? <div className="review-rec">✓ I recommend this product</div>
+          : null }
+        { review.response
           ? (
+            <div className="review-response">
+              <div className="response-header">Response:</div>
+              <p />
+              <div className="response-text">{review.response}</div>
+            </div>
+          )
+          : null }
+        <div className="review-footer">
+          <span>Helpful? </span>
+          <span>
             <button
-              className="show-more-button"
+              className="review-footer-button"
               type="button"
-              onClick={() => setFullBody(true)}
-            >SHOW MORE
+              onClick={handleHelpfulness}
+            >Yes {`(${review.helpfulness})`}
```

Comment:
Not a merge blocker. Just by switching between sort options, I am able to set something as helpful multiple times.

Reply:
Hm, that is a good point - I have sortType set as a state, so every time it changes, it re-renders the list, as I'm using the sort parameter of the API. I'll probably have to look in how to cache the data. Thoughts?